Reports/progress_report.md

```diff
@@ -0,0 +1,296 @@
+
+#### Definition of Done
+There are a number of self evlauaitons completed, there is feedback on the spec repository and, ideally, there is an active WG within the TRE Community continuing to work on it.
+
+#### Progress
+Work is folded into the [UK TRE Community](https://github.com/alan-turing-institute/data-safe-haven-team/issues/52), already SATRE is formally becoming a UK TRE Community WG.
+
+## Community building
+Creating resources for all stakeholders (inc. Citizens) to engage in the TRE conversation
+Creating and maintaining open and active communication spaces & workspaces (Slack, GH)
+Identifying and documenting everything that can be openly documented
+
+### Stakeholder landscape review
+https://github.com/alan-turing-institute/data-safe-haven-team/issues/30
+
+Contributes to:
+- Creating resources for all stakeholders (inc. Citizens) to engage in the TRE conversation
+- Creating and maintaining open and active communication spaces & workspaces (Slack, GH)
+
+#### Goal
+Across a lot of our work (DSH project, SATRE, UK TRE community) there has been a lot of discussion around who the impacted parties are, how they are categorised, what their interests/needs are etc.
+
+An effective stakeholder map showing all parties we think we should engage with will help us prioritise who to collaborate with, and strengthen our work in community building within the TRE space (which is kind of where this project is heading, above and beyond getting others to use the DSH).
+
+- [ ] Brainstorm and identify potential stakeholder groups
+- [ ] Engage different groups through interviews/workshops to better understand them
+- [ ] Create engagement pipeline & priority for different groups
+
+#### Definition of Done
+- When we have an intended end-output from engagement with our established groups (e.g. by the end of the project, we want X group to be part of the UK TRE community, we want Y group to have contributed to the DSH repo...)
+
+#### Progress
+The work done in [SATRE](https://github.com/alan-turing-institute/data-safe-haven-team/issues/23) and the [UK TRE Community](https://github.com/alan-turing-institute/data-safe-haven-team/issues/52) have directly contributed to estbalishing a relationship with key stakeholders, identify and characterise them.
+
+Direct and explicit work on this story have not been carried out (not scheduled in this period).
+
+We are now establishing the engagement pipleline by creating an internal CRM (sharepoint based currently).
+
+### UK TRE Community leadership
+https://github.com/alan-turing-institute/data-safe-haven-team/issues/52
+
+Contributes to:
+- Creating resources for all stakeholders (inc. Citizens) to engage in the TRE conversation
+- Creating and maintaining open and active communication spaces & workspaces (Slack, GH)
+
+#### Goal
+Provide a space for those involved in building, using and responsible for governance of TREs to discuss and reccomend best practices.
```

Comment:
```suggestion
Provide a space for those involved in building, using and responsible for governance of TREs to discuss and recommend best practices.
```

I think the goal is bigger than that, right? Not just provide a space....

Reply:
_I_ would think so but I think this is a place where we haven't come to an agreement yet.

I feel the UK TRE Community WG is still trying to decide what the community should be.